arkouda/util.py

```diff
@@ -303,12 +302,15 @@ def attach(name: str, dtype: str = "infer"):
 
         return DataFrame.from_return_msg(repMsg)
     else:
-        dtype = repMsg.split()[2]
-
-        if dtype == "str":
-            return Strings.from_return_msg(repMsg)
-        else:
-            return create_pdarray(repMsg)
+        try:
```

Comment:
I really want there to be a better way to handle adding `SegArray` here than using a try except. Especially because in the majority of cases it will just be a `pdarray` and will hit that failure case.

I think it may be worth still adding the response type of to each of the returns because it would eliminate the `try/except` here.

Reply:
Added your recommendation, removed `try/except`